src/Sentry.Unity/SentryUnitySDK.cs

```diff
@@ -99,4 +99,18 @@ public void Close()
                 "Exception while releasing the lockfile on the config directory.", ex);
         }
     }
+
+    public SentryUnity.LastRunState GetLastRunState()
+    {
+        if (_options.CrashedLastRun is null)
+        {
+            _options.DiagnosticLogger?.LogDebug("The SDK does not have a 'CrashedLastRun' set. " +
+                                                "This might be due to a missing or disabled native integration.");
+            return SentryUnity.LastRunState.Unknown;
+        }
+
+        return _options.CrashedLastRun.Invoke()
+            ? SentryUnity.LastRunState.Crashed
+            : SentryUnity.LastRunState.DidNotCrash;
```

Comment:
We could cache it on some `_options.InternalCrashedLastRun` when setting up the native SDK scope sync instead so users don't incur the cost of calling into the native layer.